src/elektra/redshift.ini

```diff
@@ -0,0 +1,308 @@
+check/enum/#4 = list
+description = The location provider to be used. By default, one of the supported location providers on the current OS will be chosen automatically. The provider is used to establish whether it is currently daytime or nighttime.
+default = auto
+example = geoclue2
+opt/long = location-provider
+opt/arg = required
+
+[provider/location/help]
+type = boolean
+description = Prints the help of the location providers.
+default = 0
+example = 1
+opt/long = help-providers
+opt/arg = none
+
+[provider/location/elevation/high]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds). Affects all location providers.
+default = 3.0
+example = 3.5
+opt/long = elevation-high
+opt/arg = required
+
+[provider/location/elevation/low]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds). Affects all location providers.
+default = -6.0
+example = -5.0
+opt/long = elevation-low
+opt/arg = required
+
+[provider/location/manual/lat]
+type = float
+description = The location latitude. Only applies to location provider "manual".
+check/type = float
+check/range = -90.0-90.0
+; Latitude of berlin:
+default = 52.520008
+example = 52.520008
+opt/long = lat
+opt/arg = required
+
+[provider/location/manual/lon]
+type = float
+description = The location longitude. Only applies to location provider "manual".
+check/type = float
+check/range = -180.0-180.0
+; Longitude of berlin:
+default = 13.404954
+example = 13.404954
+opt/long = lon
+opt/arg = required
+
+; Note: other location providers have no config parameters/cli arguments
+
+; BEGIN Settings used by randr and drm
+;=======================================================
+[adjustment/crtc]
+type = unsigned_short
+description = CRTC to apply adjustments to.
+default = 0
+example = 1
+opt/long = crtc
+opt/arg = required
+;=======================================================
+; END Settings used by randr and drm
+
+; BEGIN Settings used by randr and vidmode
+;=======================================================
+[adjustment/screen]
+type = unsigned_short
+description = X screen to apply adjustments to.
+default = 0
+example = 1
+opt/long = screen
+opt/arg = required
+;=======================================================
+; END Settings used by randr and vidmode
+
+; BEGIN Settings used by drm
```

Comment:
Is this needed?

Reply:
No, thanks for the hint. The fact that some adjustment methods share configuration options is actually evident from redshift's source code. Having it stated here could lead to confusion, I've removed it.